loops-and-arrays.js

```diff
@@ -35,10 +47,19 @@ const numbers = [2, 4, 5, 29, 38];
 // Zorg ervoor dat dit ook zou werken als de array wel 100 of 200 getallen zou bevatten!
 // ==========================================
 
-const squares = [30, 2, 8, 24, 11];
-
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
-// console.log(numbers) geeft: [30, 2, 8, 24, 11];
+// console.log(squares) geeft: [30, 2, 8, 24, 11];
 // Na jouw script zie je de aangepaste waardes:
-// console.log(numbers) geeft: [27000, 8, 512, 13824, 1331];
+// console.log(squares) geeft: [27000, 8, 512, 13824, 1331];
+
+const squares = [30, 2, 8, 24, 11];
+console.log(squares);
+
+for (let volume = 0; volume < squares.length; volume++) {
+    squares[volume] = squares[volume] * squares[volume] * squares[volume];
+}
```

Comment:
Je gebruikt hier geen  ' ; '  na de for loop en in de for loop wel

Reply:
Hey, als ik een ';' gebruik na mijn for loop, dan doet het programma het niet. Daar 'mag' geen ';' staan, omdat ik gebruik maak van de accolades.